UnitTests/GitUITests/UITest.cs

```diff
@@ -45,7 +45,14 @@ void HandleApplicationIdle(object sender, EventArgs e)
                 }
                 finally
                 {
-                    dialog.Close();
+                    try
+                    {
+                        dialog.Close();
+                    }
+                    finally
+                    {
+                        dialog.Dispose(); // lets RevisionDataGridView wake up and terminate its background thread(s)
```

Comment:
📝 Form.Close() already calls Dispose(). If you find that this is not occurring for a test, then it's a bug in the way we're running the message pump.

Reply:
OK, thank you for the clarification. I'll remove it and watch whether `Dispose` is called in time.